SpringAwwCrud/src/main/java/com/theironyard/AwwCrudController.java

```diff
@@ -0,0 +1,123 @@
+package com.theironyard;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpSession;
+import java.util.List;
+
+/**
+ * Created by Nigel on 7/21/16.
+ */
+
+@Controller
+public class AwwCrudController {
+    public static final String SESSION_USERNAME = "userName";
+    public static final String SESSION_PASSWORD = "password";
+
+    @Autowired
+    CarRepository cars;
+
+    @Autowired
+    UserRepository users;
+
+
+    @RequestMapping(path = "/", method = RequestMethod.GET)
+    public String home(Model model, HttpSession session) throws PasswordStorage.InvalidHashException, PasswordStorage.CannotPerformOperationException {
+
+        String userName = (String) session.getAttribute(SESSION_USERNAME);
```

Comment:
Why are these pulling from the session?  The username is in the session but `/login` is the only thing that should be pulling password.